### ISSUE TITLE
Drop Down only showing Blizzard saved mogs, no way to update or delete

### ISSUE THREAD

**Issue by Ketrel** [SLOKnightfall/BetterWardrobe#457]
As of 5.0.3

![image](https://github.com/user-attachments/assets/3723dc7b-2fa8-447f-9ec8-22434a5619e7)

The drop down is only showing the Blizzard saved mogs, not the additional ones the addon saves.  Since you can't delete from the preview panel (one of the things I requested earlier #441 ), there's no way to update or save the addon mogs

**Comment by Ketrel** [comment]
May be related to this error

```
6x BetterWardrobe/Modules/Wardrobe.lua:7452: attempt to index global 'BetterWardrobeOutfitDropDown' (a nil value)
[string "@BetterWardrobe/Modules/Wardrobe.lua"]:7452: in function `SelectSet'
[string "@BetterWardrobe/Modules/Wardrobe.lua"]:3865: in function <BetterWardrobe/Modules/Wardrobe.lua:3863>

Locals:
self = Frame {
 ModelR2C2 = DressUpModel {
 }
 ShadowCornerTopRight = Texture {
 }
 ModelR1C2 = DressUpModel {
 }
 SELECTED_SOURCE_INDEX = 3
 ShadowCornerLeft = Texture {
 }
 BGCornerFilagreeBottomLeft = Texture {
 }
 ShadowCornerRight = Texture {
 }
 NUM_ROWS = 2
 BGCornerBottomLeft = Texture {
 }
 tooltipModel = DressUpModel {
 }
 BGCornerTopRight = Texture {
 }
 ModelR1C1 = DressUpModel {
 }
 ShadowCornerBottomLeft = Texture {
 }
 Bg = Texture {
 }
 OverlayShadowBottomLeft = Texture {
 }
 APPLIED_SOURCE_INDEX = 1
 NineSlice = Frame {
 }
 searchType = 3
 OverlayShadowLeft = Texture {
 }
 ModelR2C3 = DressUpModel {
 }
 ShadowCornerTopLeft = Texture {
 }
 BackgroundTile = Texture {
 }
 BGCornerBottomRight = Texture {
 }
 OverlayShadowTop = Texture {
 }
 PendingTransmogFrame = Frame {
 }
 PagingFrame = Frame {
 }
 OverlayShadowTopRight = Texture {
 }
 OverlayShadowRight = Texture {
 }
 BGCornerTopLeft = Texture {
 }
 BGCornerFilagreeBottomRight = Texture {
 }
 PAGE_SIZE = 8
 pendingRefresh = true
 selectedSetID = 5117
 ModelR2C4 = DressUpModel {
 }
 sourceQualityTable = <table> {
 }
 ShadowCornerBottom = Texture {
 }
 layoutType = "InsetFrameTemplate"
 OverlayShadowBottom = Texture {
 }
 ModelR2C1 = DressUpModel {
 }
 ModelR1C4 = DressUpModel {
 }
 ShadowCornerTop = Texture {
 }
 ModelR1C3 = DressUpModel {
 }
 OverlayShadowTopLeft = Texture {
 }
 NUM_COLS = 4
 RightClickDropDown = BetterWardrobeSetsTransmogModelRightClickDropDown {
 }
 NoValidSetsLabel = FontString {
 }
 ShadowCornerBottomRight = Texture {
 }
 Models = <table> {
 }
 ShadowLineBottom = Texture {
 }
 ShadowLineTop = Texture {
 }
 OverlayShadowBottomRight = Texture {
 }
}
setID = 5117
(*temporary) = nil
(*temporary) = 5117
(*temporary) = <table> {
 1 = <table> {
 }
 2 = <table> {
 }
 3 = <table> {
 }
 4 = <table> {
 }
 5 = <table> {
 }
 6 = <table> {
 }
 7 = <table> {
 }
 8 = <table> {
 }
 9 = <table> {
 }
 10 = <table> {
 }
 11 = <table> {
 }
 12 = <table> {
 }
 13 = <table> {
 }
 14 = <table> {
 }
 15 = <table> {
 }
 16 = <table> {
 }
 17 = <table> {
 }
 18 = <table> {
 }
 19 = <table> {
 }
 20 = <table> {
 }
 21 = <table> {
 }
 22 = <table> {
 }
 23 = <table> {
 }
 24 = <table> {
 }
 25 = <table> {
 }
 26 = <table> {
 }
 27 = <table> {
 }
 28 = <table> {
 }
 29 = <table> {
 }
 30 = <table> {
 }
 31 = <table> {
 }
 32 = <table> {
 }
 33 = <table> {
 }
 34 = <table> {
 }
 35 = <table> {
 }
 36 = <table> {
 }
 37 = <table> {
 }
 38 = <table> {
 }
 39 = <table> {
 }
 40 = <table> {
 }
 41 = <table> {
 }
 42 = <table> {
 }
 43 = <table> {
 }
 44 = <table> {
 }
 45 = <table> {
 }
 46 = <table> {
 }
 47 = <table> {
 }
 48 = <table> {
 }
 49 = <table> {
 }
 50 = <table> {
 }
 51 = <table> {
 }
 52 = <table> {
 }
 53 = <table> {
 }
 54 = <table> {
 }
 55 = <table> {
 }
 56 = <table> {
 }
 57 = <table> {
 }
 58 = <table> {
 }
 59 = <table> {
 }
 60 = <table> {
 }
 61 = <table> {
 }
 62 = <table> {
 }
 63 = <table> {
 }
 64 = <table> {
 }
 65 = <table> {
 }
 66 = <table> {
 }
 67 = <table> {
 }
 68 = <table> {
 }
 69 = <table> {
 }
 70 = <table> {
 }
 71 = <table> {
 }
 72 = <table> {
 }
 73 = <table> {
 }
 74 = <table> {
 }
 75 = <table> {
 }
 76 = <table> {
 }
 77 = <table> {
 }
 78 = <table> {
 }
 79 = <table> {
 }
 80 = <table> {
 }
 81 = <table> {
 }
 82 = <table> {
 }
 83 = <table> {
 }
 84 = <table> {
 }
 85 = <table> {
 }
 86 = <table> {
 }
 87 = <table> {
 }
 88 = <table> {
 }
 89 = <table> {
 }
 90 = <table> {
 }
 91 = <table> {
 }
 92 = <table> {
 }
 93 = <table> {
 }
 94 = <table> {
 }
 95 = <table> {
 }
 96 = <table> {
 }
 97 = <table> {
 }
 98 = <table> {
 }
 99 = <table> {
 }
 100 = <table> {
 }
 101 = <table> {
 }
 102 = <table> {
 }
 103 = <table> {
 }
 104 = <table> {
 }
 105 = <table> {
 }
 106 = <table> {
 }
 107 = <table> {
 }
 108 = <table> {
 }
 109 = <table> {
 }
 110 = <table> {
 }
 111 = <table> {
 }
 112 = <table> {
 }
 113 = <table> {
 }
 114 = <table> {
 }
 115 = <table> {
 }
 116 = <table> {
 }
 117 = <table> {
 }
 118 = <table> {
 }
 119 = <table> {
 }
 120 = <table> {
 }
 121 = <table> {
 }
 122 = <table> {
 }
 123 = <table> {
 }
 12
```

**Comment by Aur0r4** [comment]
I'am not able to create a new set (Blizzard-Slots already full) and save it.
I think it is related to this bug too. Help D: its trial of style at the moment

**Comment by SLOKnightfall** [comment]
Fixed in the v5.0.5
